### PR TITLE
[Sanity Pass] Moving maint kinks to where they belong, while providing new and fun alternatives.

### PR DESCRIPTION
**Pull request by Feasel**
Whewww. 

This was a headache to sort through and get it all fixed but I've now officially moved the ban bait to where they belonged and restored more SS13 flavored stuff back here.

I've also introduced 2 new kegs to replace the kink chems.

Barefoot -> In honor of those who like to step on glass in maints without shoes.

Grog -> For all them rowdy greytiders who need a dark drink before busting into somewhere.

------

Introduced multiple new items to the maint loot spawn and created the chance for medical/science hud glasses to appear in maints, though rare because they're powerful.

So yay.

There will also be a healthy boost to maint pills so the old meme of maint pillery will be back in full force. 

NOTE: I did /not/ hardcore change the values for anything that was the usual. 

I only allocated the dropchance from ban bait and daki's for more interesting stuff so we have unique shit in maints and more fun "OH SHIT WHATS THIS" moments.

### CODE REVIEW

File: code/_globalvars/lists/maintenance_loot.dm [hunk 1]
```diff
@@ -106,15 +106,16 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
...
-	/obj/item/storage/pill_bottle/breast_enlargement = 2,
+	/obj/item/storage/daki = 1, //VERY IMPORTANT CIT CHANGE - adds bodypillows to maint
+	/obj/item/clothing/glasses/hud/diagnostic = 1,
+    /obj/item/clothing/glasses/hud/health = 1,
 	/obj/item/clothing/shoes/wheelys = 1,
 	/obj/item/clothing/shoes/kindleKicks = 1,
-	/obj/item/autosurgeon/penis = 1,
-	/obj/item/autosurgeon/testicles = 1,
+	/obj/item/storage/backpack/duffelbag/syndie = 1
+	/obj/item/storage/belt/fannypack =1,
```
Comment (ghost) [inline code/_globalvars/lists/maintenance_loot.dm]: missing space before the 1

Comment (jack-avery) [reply code/_globalvars/lists/maintenance_loot.dm]: wait
>/obj/item/storage/backpack/duffelbag/syndie

this might cause some confusion but the good kind.

**Comment by Ludox235** [comment]
Finally maint roaming can be fun again whithout seeing the autosurgeon (penis) for the 500th time in a row.

**Comment by dapnee** [comment]
Your spacing is incorrect and I dislike banhammers and HUD glasses in maintenance, cause spam and also giving randoms stuff science takes ages to make is bad.

**Comment by necromanceranne** [comment]
Adding syndicate duffels is gonna make a lot of people valid for maint loot and cause issues with storage powercreep being excusable. The syndicate duffels are special with good reason, and carry the risk of being spotted and arrested on the spot for flying enemy colours. Durathread duffelbags were already largely causing issues with this but at the very least those require botanists putting in some work to acquire.

**Comment by Feasel** [comment]
> Your spacing is incorrect and I dislike banhammers and HUD glasses in maintenance, cause spam and also giving randoms stuff science takes ages to make is bad.

I spent a few hours asking people what they wanted in the maint loot and a maintainer told me to put in 2 medical huds.

I decided to make it 1 and put in a seperate hud as a diagnostic.

The weight of these are the same chance as wheelie heelies spawning in maints because I replaced the lewd shit at at 1:1

**Comment by Feasel** [comment]
> Adding syndicate duffels is gonna make a lot of people valid for maint loot and cause issues with storage powercreep being excusable. Durathread duffelbags were already largely causing issues with this but at the very least those require botanists putting in some work to acquire.

As I said earlier, I was spending time asking for what to put as a replacement. I was suggested to put in the dufflebag by a maintainer so I did.

File: code/_globalvars/lists/maintenance_loot.dm (hunk 1 above)
Comment (Feasel) [reply code/_globalvars/lists/maintenance_loot.dm]: Turned out fannypacks couldn't work on maintenance loot even when fixed.

/shrug

**Comment by Trilbyspaceclone** [comment]
@Feasel Please use changlog's

**Comment by Feasel** [comment]
CHANGELOG:

Removed all sex items from maints. 

Their drop chances have been replaced with the following items; 	/obj/item/clothing/glasses/hud/diagnostic = 1,
/obj/item/clothing/glasses/hud/health = 1,
/obj/item/clothing/shoes/wheelys = 1,
/obj/item/clothing/shoes/kindleKicks = 1,
/obj/item/storage/backpack/duffelbag/syndie = 1,
/obj/item/banhammer = 1,
----
Dakimakura pillows in maints have been lowered down to 1 weight instead of 3.
    -The missing 2 weight was allocated to maint pills.

----

Maint pills are now at a weight of 4 from a weight of 1.
   -2 from Daki's. 1 from organ shit.

---

Aphro kegs in maints have been moved to kinkmate.

Their slots have been replaced with grog + barefoot kegs instead.

**Comment by necromanceranne** [comment]
What duffelbags did they specifically suggest? Was it actually the syndicate duffels because if so that was a really bad suggestion.

**Comment by silicons** [comment]
the duffle (you get from maint, so i'm implying you should make a new subtype dufflebag that doesn't hold as much as the real syndie one) should be one extra slot at most tbvqh.

**Comment by Feasel** [comment]
Gonna be closing this PR for now to probably port a more indepth system.

**Pull request closed without merging**